integration_tests/tests/apply/risk.cy.ts

```diff
@@ -0,0 +1,39 @@
+import paths from '../../../server/paths/apply'
+import { applicationFactory, personFactory } from '../../../server/testutils/factories/index'
+
+context('Risks', () => {
+  const person = personFactory.build({})
+
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubSignIn')
+    cy.task('stubAuthUser')
+  })
+
+  beforeEach(() => {
+    // Given I am logged in
+    cy.signIn()
+    // and there is Oasys data
+    cy.fixture('oasysSections.json').then(riskData => {
+      cy.task('stubOasysSections', {
+        crn: person.crn,
+        oasysSections: riskData,
+      })
+    })
+  })
+
+  describe('when I click the save and continue button', () => {
+    it('creates a new application', () => {
+      const application = applicationFactory.build({})
+      cy.task('stubCreateApplication', { application })
+
+      // given I am on the risks page
+      cy.visit(paths.applications.show({ crn: person.crn }))
+
+      // when I click the save and continue button
+      cy.get('button').click()
+
+      // then I should be redirected back to the new page
```

Comment:
are we missing this, or are we able to check that it's gone to the right page?

Reply:
good spot - I'm just adding this